### DISCUSSION TITLE
Request for Student Developer Pack

### DISCUSSION THREAD

**Discussion by bmalouli** [community/community#112531] [GitHub Education]
### Select Topic Area

Bug

### Body

Dear GitHub Support,

I hope this message finds you well. I am a student at Concordia University and I am eager to access the Student Developer Pack. However, I am encountering difficulties in processing my application even after uploading my student ID. I have already verified my university email address within my account.

I have included my student ID in this email and kindly request your assistance in expediting the approval process for the Student Pack. This would enable me to begin working promptly. Your prompt attention to this matter would be greatly appreciated.

Thank you for your understanding and assistance.

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @bmalouli , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Comment by samus-aran** [comment]
Hi @bmalouli 👋 

All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants.

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.
